GVRf/Framework/framework/src/main/java/org/gearvrf/GVRActivityBase.java

```diff
@@ -123,6 +121,7 @@ public void run() {
     abstract GVRActivityNative makeActivityNative();
     abstract GVRViewManager makeViewManager(GVRXMLParser xmlParser);
     abstract GVRMonoscopicViewManager makeMonoscopicViewManager(GVRXMLParser xmlParser);
+    abstract GoogleVRViewManager makeGoogleVRViewManager(GVRXMLParser xmlParser);
```

Comment:
GVRConfigurationManager is a file from backend_oculus, should it even be referenced here?